src/formatDate.js

```diff
@@ -8,7 +8,29 @@
  * @returns {string}
  */
 function formatDate(date, fromFormat, toFormat) {
-  // write code here
+  const separator = fromFormat.pop();
```

Comment:
The use of `pop()` on `fromFormat` modifies the original array, which might not be intended if `fromFormat` is used elsewhere. Consider using `slice()` or `spread operator` to avoid mutating the input array.